src/custom/constants/tokens/index.ts

```diff
@@ -49,6 +52,42 @@ export const V_COW: Record<number, Token> = {
   [SupportedChainId.RINKEBY]: V_COW_TOKEN_RINKEBY,
 }
 
+/**
+ * Cow token
+ */
+const COW_TOKEN_MAINNET = new Token(
```

Comment:
In that case, we should update everything else to be consistent.
Which will be a pain in the ass twice, more so on my ass as this will create tons of conflicts with the uni-merge.

Or we can do it just for COW and later thing about vCOW...

Reply:
I will take a note and we can fix this after the uni merge